android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DDMStructure.java

```diff
@@ -60,10 +64,33 @@ public DDMStructure createFromParcel(Parcel in) {
     private Long classNameId;
     private String classPK;
 
+    private List<FormPage> pages;
```

Comment:
why not new ArrayList to not have to check for null values?

Reply:
Good point, I will add it.